wrappers/javascript/packages/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -617,6 +627,380 @@ export class NodeJSAnoncreds implements Anoncreds {
     return new ObjectHandle(handleReturnPointer<number>(ret))
   }
 
+  public createW3CCredentialOffer(options: {
+    schemaId: string
+    credentialDefinitionId: string
+    keyCorrectnessProof: ObjectHandle
+  }): ObjectHandle {
+    const { schemaId, credentialDefinitionId, keyCorrectnessProof } = serializeArguments(options)
+
+    const ret = allocatePointer()
+    this.nativeAnoncreds.anoncreds_create_w3c_credential_offer(
+      schemaId,
+      credentialDefinitionId,
+      keyCorrectnessProof,
+      ret
+    )
+    this.handleError()
+
+    return new ObjectHandle(handleReturnPointer<number>(ret))
+  }
+
+  public createW3CCredentialRequest(options: {
```

Comment:
Same here: `W3c`

and also, I don't see a reason for a w3c credential request, only the end result is different I think?